### PR TITLE
Update transformer_asr.py

### PR DESCRIPTION
**Pull request by MeysamMoghaddam**
Add small epsilon (1e-10) to stddevs to prevent division by zero

Modified the normalization step by adding a small constant to stddevs  to avoid potential division by zero errors when standard deviation is zero.

### CODE REVIEW

File: examples/audio/transformer_asr.py [hunk 1]
```diff
@@ -372,7 +372,7 @@ def path_to_audio(path):
     # normalisation
     means = tf.math.reduce_mean(x, 1, keepdims=True)
     stddevs = tf.math.reduce_std(x, 1, keepdims=True)
-    x = (x - means) / stddevs
+    x = (x - means) / (stddevs + 1e-10)
```
Comment (mattdangerw) [inline examples/audio/transformer_asr.py:375]: does using keras.epsilon work? and can you add the generated files?
